qiskit_sphinx_theme/sidebar.html

```diff
@@ -0,0 +1,88 @@
+<nav data-toggle="wy-nav-shift" class="pytorch-left-menu" id="pytorch-left-menu">
+    <div class="pytorch-menu pytorch-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
+      <div class="sidebar">
+    <div class="pytorch-left-menu-search">
+        {% include "searchbox.html" %}
+    </div>
+
+    <!-- render sidebar from the toctree -->
+    {% set global_toc = toctree(maxdepth=1,
+                                collapse=theme_collapse_navigation|tobool,
+                                includehidden=theme_includehidden|tobool) %}
+    {% if global_toc %}
+        {{ global_toc }}
+    {% endif %}
+    
+    <!-- if sidebar_headings var set, render custom sidebar layout -->
+    {% if sidebar_headings %}
+        {% for h in sidebar_headings %}
+            {% if h.subheadings %}
+                <div class="sidebar-l1-expandable">{{h.title}}</div>
+                <div class="sidebar-subheadings">
+                    {% for sub in h.subheadings %}
+                        {% if sub.url.startswith('http')%}
+                            <div class="sidebar-l2"><a class="reference external" target="_blank" rel="noopener noreferrer" href="{{sub.url}}">{{sub.title}}</a></div>
+                        {% else %}
+                            <div class="sidebar-l2"><a class="reference internal" href="{{ pathto(sub.url)}}">{{sub.title}}</a></div>
+                        {% endif %}
+                    {% endfor %}
+                </div>
+            {% else %}
+                {% if h.url.startswith('http')%}
+                    <div class="sidebar-l1"><a class="reference external" target="_blank" rel="noopener noreferrer" href="{{ h.url }}">{{h.title}}</a></div>
+                {% else %}
+                    <div class="sidebar-l1"><a class="reference internal" href="{{ pathto(h.url) }}">{{h.title}}</a></div>
+                {% endif %}
+            {% endif %}
+        {% endfor%}
+    {% endif %}
+
+    <!-- PREVIOUS RELEASES -->
+        {% if version_list %}
+            <div class="sidebar-l1-expandable">Previous Releases</div>
```

Comment:
Should we make the style for previous releases different (make it bold?) from the rest of the sidebar items to make it standout?

Reply:
hmm I'm not sure what benefit that adds. Is there a reason you think that section should be bold? Why do we want to emphasise that section more than others?

Reply:
After thinking a bit more, I think this is fine.